app/Providers/UpdateServiceProvider.php

```diff
@@ -24,7 +24,7 @@ protected function registerRoutes()
             'as'         => 'update.',
             'prefix'     => 'update',
             'middleware' => ['web', 'auth', 'ability:admin,admin-access'],
-            'namespace'  => 'Modules\Updater\Http\Controllers',
```

Comment:
These service providers (Update/Installer/Importer), just leave the routes in the main `RouteServiceProvider`, just create new methods like `RegisterInstallerRoutes()`, and do the same for the Updater/Importer. It's less files that need to be loaded on page load

Reply:
Okay.